packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -601,7 +601,8 @@ class _AnimatedPositionedState extends AnimatedWidgetBaseState<AnimatedPositione
 /// Animated version of [Opacity] which automatically transitions the child's
 /// opacity over a given duration whenever the given opacity changes.
 ///
-/// Animating an opacity is relatively expensive.
+/// Animating an opacity is relatively expensive because it requires repeatedly
+/// painting the child into an intermediate buffer every frame of the animation.
```

Comment:
Is the statement "Animating an opacity is relatively expensive." incorrect?
If it's correct, what's the reason, if not what I described?